content/en/network_monitoring/cloud_network_monitoring/guide/detecting_application_availability.md

```diff
@@ -0,0 +1,125 @@
+
+3. Click into one of the traffic paths on this page to open the side panel. This provides more detailed telemetry to help you further debug your network dependencies. 
+
+4. While on the side panel view, check the Flows tab to determine if the communication protocol is TCP or UDP, and review metrics like RTT, Jitter, and packets sent and received to help identify the cause of a high retransmit count.
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_sidepanel_flows.png" alt="Side panel of a traffic flow, highlighting the Flows tab">}}
+
+## Visibility into network traffic 
+
+Datadog CNM consolidates relevant distributed traces, logs, and infrastructure data into a single view, allowing you to identify and trace issues back to the originating request from an application.
+
+In the example below, check the Traces tab under Network Analytics to view distributed traces of requests between source and destination endpoints, which can help you pinpoint where application-level errors occur.
+
+To identify if an issue is an Application or Network issue, you can use the following steps:
+
+1. Navigate to [Infrastructure > Cloud Network > Analytics][1].  
+2. In the Summary graphs, click a line of communication that has a lot of volume and high RTT time:
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_isolate_series.png" alt="CNM analytics page, clicking on a path with high RTT Time">}}
+
+3. Click **Isolate this series**  
+4. Click into this path and click on the Flows tab to observe RTT time:
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_sidepanel_rtt.png" alt="CNM sidepanel, highlighting the RTT time column">}}
+
+5. On this page, CNM correlates network metric round-trip time (RTT) with application request latency, to help us identify if the issue is a network or application issue. In this particular example, observe that the RTT time is slightly high but has come down over time and needs to be investigated further.  
+     
+6. On this same page, click the Traces tab and investigate the Duration column: 
+
+   {{< img src="network_performance_monitoring/guide/detecting_network_insights/cnm_traces_duration.png" alt="CNM sidepanel, highlighting the Traces tab and duration column">}}
+
+   Observe that although network latency (RTT) is high, the application request latency (Duration) is normal, so in this case, the issue is likely network-related, and there's no need to investigate the app code. 
+
+   Conversely, *if network latency is stable but application latency (Duration) is high*, the problem likely stems from the app, and you can explore code-level traces by clicking on one of the service paths in the Traces tab to find the root cause, which takes you to the APM flame graph relative to this service:
```

Comment:
This is clear example stuff that I don't think really belongs in a procedure. It's good information, though!

I also wonder if you could make the connections between network latency and RTT, and application request latency and duration, higher up in the topic?

```suggestion
   Observe that although network latency (RTT) is high, the application request latency (Duration) is normal, so in this case, the issue is likely network-related, and there's no need to investigate the app code. 

   Conversely, *if network latency is stable but application latency (Duration) is high*, the problem likely stems from the app, and you can explore code-level traces by clicking on one of the service paths in the **Traces** tab to find the root cause, which takes you to the APM flame graph relative to this service:
```